Gemfile

```diff
@@ -13,8 +13,8 @@ gem "unicorn-worker-killer", "0.4.4"
 
 # Federation
 
-gem "diaspora_federation-json_schema", "0.2.0", github: "SuperTux88/diaspora_federation", branch: "json-schemas-helper"
```

Comment:
That's probably because I deleted that branch after you merged the PR. So you should use `develop` here